src/test/CryptoDashboard.test.js

```diff
@@ -0,0 +1,18 @@
+import { Provider } from 'react-redux';
```

Comment:
- [ ] Kindly add unit tests for pure functions in Redux code as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.).
- [ ] Good job on implementing the snapshot tests for components, but Kindly add integration tests using React Testing Library as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library).

Reply:
Hi @gemmen29 ,
Thanks for your time and for reviewing my code. I will apply the necessary changes as you advised. 🙏

Reply:
This change has been done well